frontend/js/protein.js

```diff
@@ -603,3 +603,111 @@ $(document).ready(() => {
 
     }
 });
+
+TourHelper.attachTour('#help-menu-item', [
+    {
+        target: '#spacedText',
+        content: 'This page shows you all about a specific protein.',
+        placement: ['bottom']
+    },
+    {
+        target: '#experiment-number',
+        content: 'Here you can switch between the SAME protein, used in different experiments.',
+        placement: ['left', 'right', 'top', 'bottom']
+    },
+    {
+        before: () => {
+            $('#experiment-number input.search').focus();
+            return new Promise((res, rej) => {
+                setTimeout(() => {res();}, 200);
+            });
+        },
+        target: '#experiment-number > div > .menu',
+        content: 'This is a list of all the experiments, in which this protein is used.',
+        placement: ['bottom'],
+        after: () => {
+            $('#experiment-number input.search').focus();
+            return Promise.resolve();
+        }
+    },
+    {
+        // before: () => { // doesnt work
+        //     $('path.highcharts-point:nth-child(3)').trigger('mouseenter');
+        //     return Promise.resolve();
+        // },
+        target: '#protein-curve',
+        content: 'The proteins melting curve from 37° to 64°. Hovering over it gives you the discrete values for the specific degree.',
```

Comment:
> This interactive graph shows the registered temperatures for the given protein in the selected experiment.

--> Don't put the exact temperature range: we might be getting data with different temperatures (btw, forgot to talk about that with you in the call today).